spec/models/collection_draft_proposal_spec.rb

```diff
@@ -210,10 +210,10 @@
     # not be possible if the other actions are successfully blocked
 
     it 'creating a draft proposal should fail to save' do
```

Comment:
do we have a test in this file that tests creating a proposal without the request_type?